README.md

```diff
@@ -540,6 +540,10 @@ These are items relevant *only* on OS X.
 
 ## Windows only
 
+These items are relevant *only* on Windows.
+
+- Since Windows 10, you can use [Bash on Ubuntu on Windows](https://msdn.microsoft.com/commandline/wsl/about), which provides a familiar Bash environment with Unix command line utilities and allows Linux programs to run on Windows.
+
```

Comment:
* On Windows 10, you can
* command line utilities. On the plus side, this allows Linux programs to run on Windows. On the other hand this does not allow Windows programs to run from the Bash prompt.